examples/resources/iam_policy/resource_iam_namespace.tf

```diff
@@ -0,0 +1,23 @@
+# Create Role Bindings for Namespace Scope
+resource "tanzu-mission-control_iam_policy" "create_iam_policy_namespace" {
+  scope {
+    namespace {
+      management_cluster_name = "attached"
+      provisioner_name        = "attached"
+      cluster_name            = "demo"
+      namespace_name          = "tf_namespace"
+    }
+  }
+
+  role_bindings {
+    role = "namespace.view"
+    subjects {
+      name = "test-1"
+      kind = "USER"
+    }
+    subjects {
+      name = "test-2"
+      kind = "GROUP"
+    }
+  }
+}
```

Comment:
the files should end with a new line. Do the changes in all the places as applicable